docker/deployment/app-deployment.yaml

```diff
@@ -100,22 +100,28 @@ spec:
       labels:
         app: app
     spec:
-      securityContext: # see https://github.com/sillsdev/web-languageforge/pull/974 for further explanation of the need for a securityContext here
-        fsGroup: 33 # this is www-data from /etc/passwd, k8s requires explicit assignment otherwise the volume will get mounted with root and www-data won't be allowed to create folders on-the-fly in there.
-        fsGroupChangePolicy: OnRootMismatch
       volumes:
       - name: assets
         persistentVolumeClaim:
           claimName: lf-project-assets
       - name: sendreceive-data
         persistentVolumeClaim:
           claimName: lfmerge-sendreceive-data
+      initContainers:
+      - name: initialize-volume-ownership
+        image: busybox:1.33
+        command: ['sh', '-c', 'mkdir -p /var/www/html/assets && chown www-data:www-data /var/www/html/assets && mkdir -p /var/lib/languageforge/lexicon/sendreceive && chown www-data:www-data /var/lib/languageforge/lexicon/sendreceive']
```

Comment:
I double-checked the `busybox:1.33` image and it does have UID and GID 33 assigned for the user and group named `www-data`. So doing this will assign UID 33 and GID 33, which is what we want.